Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -0,0 +1,1904 @@
+    fetch_time = demisto.params().get('first_fetch')
+    max_fetch = int(demisto.params().get('max_fetch', '50'))
+    alerts_states = argToList(alerts_states_to_retrieve)
+
+    # Handle first time fetch, fetch incidents retroactively
+    if not last_fetch:
+        last_fetch, _ = parse_date_range(fetch_time, date_format=DATE_FORMAT)
+
+    demisto.debug(f'Get last run: last_id {last_id}, last_time: {last_fetch}.\n')
+
+    last_fetch = parse(last_fetch)
+    current_fetch = last_fetch
+
+    url_suffix = '/plugin/products/detect3/api/v1/alerts?' + state_params_suffix(alerts_states)
+
+    raw_response = client.do_request('GET', url_suffix)
+
+    # convert the data/events to demisto incidents
+    incidents = []
+    for alarm in raw_response:
+        incident = alarm_to_incident(client, alarm)
+        temp_date = parse(incident.get('occurred'))
+
+        # update last run
+        if temp_date > last_fetch:
+            last_fetch = temp_date
+
+        # avoid duplication due to weak time query
+        if temp_date > current_fetch and (new_id := alarm.get('id', last_id)) > last_id:
+            incidents.append(incident)
+            last_id = new_id
+
+        if len(incidents) >= max_fetch:
+            break
+
+    demisto.setLastRun({'time': datetime.strftime(last_fetch, DATE_FORMAT), 'id': str(last_id)})
+
+    demisto.debug(f'Set last run: last_id {last_id}, last_time: {last_fetch}.\n')
+    demisto.debug(f'Fetched {len(incidents)} incidents.')
+
+    return demisto.incidents(incidents)
+
+
+''' INTEL DOCS COMMANDS FUNCTIONS '''
+
+
+def get_intel_doc(client: Client, data_args: dict) -> Tuple[str, dict, Union[list, dict]]:
+    """ Gets a single intel doc from a given id.
+
+        :type client: ``Client``
+        :param client: client which connects to api.
+        :type data_args: ``dict``
+        :param data_args: request arguments.
+
+        :return: human readable format, context output and the original raw response.
+        :rtype: ``tuple``
+
+    """
+    id_ = data_args.get('intel-doc-id')
+    try:
+        raw_response = client.do_request('GET', f'/plugin/products/detect3/api/v1/intels/{id_}')
+    # If the user provided a intel doc ID which does not exist, the do_request will throw HTTPError exception
+    # with a "Not Found" message.
+    except requests.HTTPError as e:
+        if 'not found' in str(e):
+            raise DemistoException(f'Please check the intel doc ID and try again.\n({str(e)})')
+    intel_doc = get_intel_doc_item(raw_response)
+    # A more readable format for the human readble section.
+    if intel_doc:
+        intel_doc['LabelIds'] = str(intel_doc.get('LabelIds', [])).strip('[]')
+    context_data = format_context_data(raw_response)
+    context = createContext(context_data, removeNull=True)
+    outputs = {'Tanium.IntelDoc(val.ID && val.ID === obj.ID)': context}
+
+    human_readable = tableToMarkdown('Intel Doc information', intel_doc, headers=list(intel_doc.keys()),
```

Comment:
the current value of `headers` is the default behavior of `tableToMarkdown` - please provide a hard-coded ordered list (here and in similar cases in other commands)